src/network/message_blockdata.rs

```diff
@@ -35,6 +35,8 @@ pub enum Inventory {
     Transaction(Txid),
     /// Block
     Block(BlockHash),
+    /// Witness Transaction by Wtxid
+    WTx(Wtxid),
```

Comment:
SGTM, Core doesn't have a better one, but at least they have a prefix.

Reply:
Since the other one is `Transaction`, shouldn't this one be `WitnessTransaction` for consistency? I mean I agree that having it shorter like this is nicer :) Yeah perhaps just keep it :)